### DISCUSSION TITLE
Servers List Vanishes now

### DISCUSSION THREAD

**Discussion by bobcozzi** [codefori/vscode-ibmi#2274] [Q&A] [answered]
Once I expand and connect to my local IBM i server, the "SERVERS" vanish and I have no way to go back and open another Server. This is a relatively new "feature" so I'm wondering if I did something or if this is expected?

**Comment by sebjulliand** [comment, marked as answer]
This is expected, yes. The Servers view used to vanish only after the connection was successful. Now thanks to this "feature" it's hidden as soon as you connect. So it gets hidden sooner now, that's the main difference.
You actually never had a way to go back and select another server once connected.

**Comment by bobcozzi** [comment]
That's, well, unexpected.

**Reply by sebjulliand** [reply]
How so? 😅

**Reply by bobcozzi** [reply]
I work on multiple partitions at the same time all the time.
So now I would have to exit CODE and relaunch it to connect to another/different partition.

**Reply by sebjulliand** [reply]
Nope, you just need to disconnect. The icon is at the bottom, in the status bar:
![image](https://github.com/user-attachments/assets/961692a8-c2ac-4400-b509-7f852322ffbe)

You can also have multiple VS Code open at the same time; it might be easier for you to switch between LPARs.

**Reply by bobcozzi** [reply]
Thanks. I thought I had looked at that icon for a disconnect but must have overlooked it. This is slightly easier. And I'd prefer not to have multiple sessions open. But I'll try to work with it as is. Thanks for the insight.
